mediaqueries-5/Overview.bs

```diff
@@ -478,6 +478,36 @@ Detecting the desire for less motion on the page: the 'prefers-reduced-motion' f
 
 	</dl>
 
+<h3 id="prefers-reduced-transparency">
+Detecting the desire to reduce the use of transparent or translucent layering effects on the page: the 'prefers-reduced-transparency' feature</h3>
```

Comment:
Okay, but clearly many UI effects nowadays combine a certain levels of opacity/transparency with certain levels of translucency (see example graphics on https://en.wikipedia.org/wiki/Transparency_and_translucency) and as long as this MQ should detect a combined preference then transparency for the name and translucency as part of the docs seems reasonable. My guess is still that apple picked "transparency" as that is an easy concept to get to every user.

`opacity` is btw the key word in CSS used to describe (non boolean shades of) *transparency of layers*, not transparency.